pkg/operator/object/ceph/rgw.go

```diff
@@ -348,7 +348,6 @@ func rgwContainer(store rookalpha.ObjectStore, version string) v1.Container {
 		Args: []string{
 			"rgw",
 			fmt.Sprintf("--config-dir=%s", k8sutil.DataDir),
-			fmt.Sprintf("--rgw-name=%s", store.Name),
```

Comment:
This flag should still be set, but to the value of `rgwName` as here in the flag https://github.com/rook/rook/pull/1484/files#diff-1b8cf32eef511abe83fae3ce2b907113R44.

Reply:
The value of the said `rgwName` is unknown since this is the operator's scope, not the pod one. Can you please clarify this comment.